src/components/ProgressIndicator/ProgressIndicator.stories.js

```diff
@@ -24,10 +24,10 @@ export default {
       {
         to: 'first-step',
         label: 'Stap 1',
-        isCompleted: true,
-        isApplicable: true,
+        isCompleted: false,
+        isApplicable: false,
         isCurrent: false,
-        canNavigateTo: true,
+        canNavigateTo: false,
```

Comment:
These were modified in order to show that a non-applicable step is not navigable